client/app/bundles/course/assessment/question/scribing/components/SingleFileInput.jsx

```diff
@@ -0,0 +1,186 @@
+import React from 'react';
```

Comment:
Actually why did you copy this file from the `SingleFileInput` component from redux form?

Reply:
Chatted with @nicholette offline and suggested to modify `SingleFileInput` with some reusability for this use case.